rundeckapp/grails-app/controllers/rundeck/controllers/WorkflowController.groovy

```diff
@@ -36,6 +38,8 @@ import javax.servlet.http.HttpServletResponse
 
 class WorkflowController extends ControllerBase {
     def frameworkService
+    AppAuthContextEvaluator rundeckAuthContextEvaluator
+    AuthContextProvider rundeckAuthContextProvider
```

Comment:
Since these two beans are a common requirement in pretty much anything that needs to do authentication, should they be combined in an interface so that only 1 bean injection is needed?

Reply:
i'm doing some cleanup along this line